treeherder/auth/backends.py

```diff
@@ -102,21 +77,14 @@ def authenticate(self, auth_header=None, host=None, port=None):
         # Look for an existing user in this order:
         #
         # 1. Matching username/clientId
-        # 2. Matching email.
```

Comment:
It would be more appropriate to re-word the comment.  Something like:

    Look for an existing user by username/clientId.  If not found, then create it as long
    as it has an email.